microsoft/testsuites/vm_extensions/waagent.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import uuid
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.sut_orchestrator.azure.features import AzureExtension, ExtensionNames
+
+
+@TestSuiteMetadata(
+    area="azure",
+    category="functional",
+    description="BVT for VM Agent",
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class WaAgentBvt(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        Runs an extension and verifies it executed on the remote machine.
+        """,
+        priority=1,
+        requirement=simple_requirement(supported_features=[AzureExtension]),
+    )
+    def verify_vm_agent(self, log: Logger, node: Node) -> None:
+        # Any extension will do, use CustomScript for convenience.
+        # Use the extension to create a unique file on the test machine.
+        test_file = "/tmp/{0}".format(uuid.uuid4())
+        script = "touch {0} && echo Created {0}".format(test_file)
+        settings = {"commandToExecute": script}
+        extension = node.features[AzureExtension]
+        result = extension.create_or_update(
+            name="CustomScript",
+            publisher="Microsoft.Azure.Extensions",
+            type=ExtensionNames.custom_script,
```

Comment:
I removed the enumeration. please use string directly.

Reply:
ok, have you merged your change? otherwise the tools will complain. i can make the change once you merge